nemo/collections/nlp/modules/common/megatron/token_level_encoder_decoder.py

```diff
@@ -725,6 +725,7 @@ def __init__(self, *args, **kwargs):
         self.speech_head_type = "token_level"
         self.attn_prior_scaledown_start_step = 10000
         self.attn_prior_end_step = 11000
+        self.use_alignment_loss = False
```

Comment:
Why does this default to False?

Reply:
i tried to keep consistent defaults by following the same default setup at `Line 201:nemo/collections/tts/models/speechllm/megatron_t5_speechllm_model.py`. 

```python
if cfg.get('use_alignment_loss', False):
```